lvmd/vgservice_test.go

```diff
@@ -183,12 +183,12 @@ func testVGService(t *testing.T, vg *command.VolumeGroup) {
 		t.Errorf("Free bytes mismatch: %d, expected: %d, freeBytes: %d", res2.GetFreeBytes(), expected, freeBytes)
 	}
 
-	_, err = vg.CreateVolume("test3", 1<<30, nil, 2, "4k")
+	_, err = vg.CreateVolume("test3", 1<<30, nil, 2, "4k", nil)
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	_, err = vg.CreateVolume("test4", 1<<30, nil, 2, "4M")
+	_, err = vg.CreateVolume("test4", 1<<30, nil, 2, "4M", nil)
```

Comment:
Please also add a test case for `lvcreateOptions != nil` case.

Reply:
Added, but it looks like it is always skipping these tests in the CI? I added an issue for it: #438